text/0000-core-builder.md

```diff
@@ -0,0 +1,46 @@
+
+Users will be able to use the new builder to build and run their applications using any of the language family buildpacks included. Instead of requireing a decision for an optimised builder, they can "just" start and later decide on an optimised `run` image.
+
+## Motivation
+
+The Paketo project will be easier to use for users - they can start without being asked to decide on optimisation.
+The stacks and builders maintainer team will have less work to do - they will only need to maintain a single stack and builder instead of three.
+
+## Detailed Explanation
+
+We would replace the current `base`, `tiny` and `static` stacks with a single stack that includes all packages required by the buildpacks. Similarly, we would replace the current `base`, `tiny` and `static` builders with a single builder.
+
+## Rationale and Alternatives
+
+We could leave everything as it is. We would stick to maintaining four stacks repositories and eight builder repositories and creating new ones for every LTS we adopt. But that would increase the burden on the maintainers.
+
+## Implementation
+
+I would propose to create the new stack and builder as part of moving to Ubuntu Noble Numbat. I.e. 
+- we should create a repository `paketo-buildpacks/stack-ubuntu-noble`
+- we should take the `build` image content from the `paketo-buildpacks/noble-base-stack` repository and apply it to `paketo-buildpacks/stack-ubuntu-noble`
+- we should take the `run` image content from the `paketo-buildpacks/noble-base-stack` repository and apply it to `paketo-buildpacks/stack-ubuntu-noble`
+- we should take the `run` image content from the `paketo-buildpacks/noble-tiny-stack` repository and apply it to `paketo-buildpacks/stack-ubuntu-noble` as additional run image
+- we should take the `run` image content from the `paketo-buildpacks/noble-static-stack` repository and apply it to `paketo-buildpacks/stack-ubuntu-noble` as additional run image
+- we should create a repository `paketo-buildpacks/builder-ubuntu-noble` and another`paketo-buildpacks/builder-ubuntu-noble-buildpackless`
+    - we should use the above `build` image as a base layer for the `builder` image
+    - we should use the above `run` image as the default `run` image for the `builder.toml`
+    - we should use the above `run-static` and `run-tiny` images as additional `run` images for the `builder.toml`
+
+I would propose to delete the `paketo-buildpacks/noble-base-stack`, `paketo-buildpacks/noble-tiny-stack` and `paketo-buildpacks/noble-static-stack` repositories.
+I would propose to delete the `paketo-buildpacks/builder-noble-buildpackless-static`, `paketo-buildpacks/builder-noble-buildpackless-tiny` and `paketo-buildpacks/builder-noble-buildpackless-base` repositories.
+
+## Prior Art
+
+None.
+
+## Unresolved Questions and Bikeshedding
+
```

Comment:
Q: What do we do about stack/target & builder metadata?

I think it would be nice if the proposal described the new metadata to use for the stack/target & builder. Or is the plan to continue on using the existing metadata?  I don't have a strong opinion, but I think we should specify what that will be so it's clear.

Reply:
Minimally, I think the RFC should propose the builder & stack ids to use. The other builder & stack metadata is not as important.

I think we should start adding target metadata, if we're not doing that already. My understanding is that it can live side-by-side with stack metadata. So we should probably specify in the RFC what target metadata to set, see https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md#base-image-metadata.

I'd also like to see included the specific current image references and the future image reference.

I think this will help with implementation, but I think it'll also just make it more clear how we are consolidating things. So maybe have it in a before/after style?

Reply:
> Minimally, I think the RFC should propose the builder & stack ids to use. The other builder & stack metadata is not as important.

Actually, we might even want to avoid stack id and stack metadata in general and switch over to using the new metadata. Are there many things to consider other that the few explicit references to the tiny stack id? Might be worth tackling as part of this.

Reply:
> I'd also like to see included the specific current image references and the future image reference.
> 
> I think this will help with implementation, but I think it'll also just make it more clear how we are consolidating things. So maybe have it in a before/after style?

I added https://github.com/paketo-buildpacks/rfcs/pull/313/files#diff-a6cd29e680d46138ca93c24308bc8ac8f126e1d73463d1691e5091c2faf11eadR42-R48